ldb/urls.py

```diff
@@ -40,5 +33,6 @@
     path('organizations/create/', OrganizationEditView.as_view(), name='ldb_organizations_create'),
     path('committees/', CommitteeMembershipFilterView.as_view(), name='ldb_committees'),
 
-    path('export/', ExportView.as_view(), name="ldb_export"),
+    path('export/', ExportView.as_view(), name='ldb_export'),
+    path('export/exec', export.Export.as_view(), name='ldb_export_execute'),
```

Comment:
It feels as if this should be the same view (one with `GET`, other `POST`) . Also, convention is to have a trailing slash in the urls.

Reply:
You should be able to overwrite the `get()` and `post()` in the `APIView`. I believe when you return a `HttpResponse` in the get you can return a rendered template, the post should then return what the `APIView` currently does. But maybe we should just leave that for now.

As for the trailing slash convention. Django has an [`APPEND_SLASH`](https://docs.djangoproject.com/en/dev/ref/settings/#append-slash) setting which defaults to `True`:
>When set to True, if the request URL does not match any of the patterns in the URLconf and it doesn’t end in a slash, an HTTP redirect is issued to the same URL with a slash appended.

From which I infer that having trailing slashes is the convention. But mostly it is that the rest of this project has trailing slashes, so for this project at least it is the convention.

Reply:
Done!